examples/locale/main.rs

```diff
@@ -0,0 +1,70 @@
+//! Example showing how to load a Locale file as an Asset using the Loader.
+
+extern crate amethyst;
+
+use amethyst::assets::*;
+use amethyst::locale::*;
+use amethyst::prelude::*;
+
+struct Example {
+    handle: Option<Handle<Locale>>,
+}
+
+impl Example {
+    pub fn new() -> Self {
+        Example {
+            handle: None,
+        }
+    }
+}
+
+impl State for Example {
+    fn on_start(&mut self, world: &mut World) {
+        world.add_resource(AssetStorage::<Locale>::new());
```

Comment:
No longer needed, you add `Processor<Locale>` below, which will register this resource.

Reply:
Ah, this is not fixed yet.

Reply:
This should be using `world.exec`.